probes/ssh_passwd/main.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+IP=$1
+FOLDER_PATH=$2
+
+source /etc/secant/probes.conf
+
+hydra -L $SSH_PASSWD/user.list -P $SSH_PASSWD/passwd.list -t 8 $IP ssh > $FOLDER_PATH/hydra.stdout 2>$FOLDER_PATH/ssh_passwd.stderr
+grep -q "0 valid" $FOLDER_PATH/hydra.stdout
+if [ "$?" -eq "0" ]; then
+    echo OK
+    echo Password was not cracked
+    echo Dictionary attack finished successfully with 0 valid passwords found.
+    exit 0
+else
```

Comment:
this else is useless (the code exits in the upper branch), please remove it (it'll simplify the code)